cmd/andromeda-liquid-server/main.go

```diff
@@ -188,37 +188,37 @@ func (l *liquidLogic) ScanUsage(ctx context.Context, projectUUID string, req liq
 		InfoVersion: 1,
 		Resources: map[liquid.ResourceName]*liquid.ResourceUsageReport{
 			"datacenters": {
-				Quota: resp.Payload.Quota.Datacenter,
+				Quota: option.Some(*resp.Payload.Quota.Datacenter),
```

Comment:
If the dereference is safe, then this is fine. If the pointer might be nil, you can use options.FromPointer:
```suggestion
				Quota: options.FromPointer(resp.Payload.Quota.Datacenter),
```